registry/remote/example_test.go

```diff
@@ -45,9 +45,44 @@ const (
 )
 
 var (
-	exampleLayerDigest    = digest.FromBytes([]byte(exampleLayer)).String()
-	exampleManifestDigest = digest.FromBytes([]byte(exampleManifest)).String()
-	exampleRefereceDigest = "sha256:b2122d3fd728173dd6b68a0b73caa129302b78c78273ba43ead541a88169c855"
+	exampleLayerDigest        = digest.FromBytes([]byte(exampleLayer)).String()
+	exampleManifestDigest     = digest.FromBytes([]byte(exampleManifest)).String()
+	exampleManifestDescriptor = artifactspec.Descriptor{
+		MediaType: ocispec.MediaTypeImageManifest,
+		Digest:    digest.Digest(exampleManifestDigest),
+		Size:      int64(len(exampleManifest))}
+	exampleRefereceDigest       = "sha256:b2122d3fd728173dd6b68a0b73caa129302b78c78273ba43ead541a88169c855"
+	exampleReferenceManifest, _ = json.Marshal(artifactspec.Manifest{
+		MediaType:    artifactspec.MediaTypeArtifactManifest,
+		ArtifactType: "example reference manifest",
+		Subject:      &exampleManifestDescriptor})
+	exampleReferenceManifestDescriptor = artifactspec.Descriptor{
+		MediaType:    artifactspec.MediaTypeArtifactManifest,
+		ArtifactType: "example reference manifest",
+		Digest:       digest.FromBytes(exampleReferenceManifest),
+		Size:         int64(len(exampleReferenceManifest))}
+	exampleSignatureManifest, _ = json.Marshal(artifactspec.Manifest{
+		MediaType:    artifactspec.MediaTypeArtifactManifest,
+		ArtifactType: "example/signature",
+		Subject:      &exampleManifestDescriptor})
+	exampleSignatureManifestDescriptor = artifactspec.Descriptor{
+		MediaType:    artifactspec.MediaTypeArtifactManifest,
+		ArtifactType: "example/signature",
+		Digest:       digest.FromBytes(exampleSignatureManifest),
+		Size:         int64(len(exampleSignatureManifest))}
+	exampleSBoMManifest, _ = json.Marshal(artifactspec.Manifest{
+		MediaType:    artifactspec.MediaTypeArtifactManifest,
+		ArtifactType: "example/SBoM",
+		Subject:      &exampleManifestDescriptor})
+	exampleSBoMManifestDescriptor = artifactspec.Descriptor{
+		MediaType:    artifactspec.MediaTypeArtifactManifest,
+		ArtifactType: "example/SBoM",
+		Digest:       digest.FromBytes(exampleSBoMManifest),
+		Size:         int64(len(exampleSBoMManifest))}
```

Comment:
There are too many of these for the example. I think we just need the sbom one and the signature one.

Reply:
Deleted the extra one.